tests/integration/test_runs.py

```diff
@@ -0,0 +1,87 @@
+import json
+
+from tests.constants import USER_ID_FOR_TESTING
+
+from finetuner.client.client import Client
+
+
+def test_runs(run_config, experiment_name='exp', first_run='run1', second_run='run2'):
+    # create a client
+    client = Client(user_id=USER_ID_FOR_TESTING)
+    # create an experiment and retrieve it
+    client.create_experiment(experiment_name)
+    response = client.get_experiment(name=experiment_name).json()
+    assert response['name'] == experiment_name
+    assert response['status'] == 'ACTIVE'
+    # create a first run
+    client.create_run(
+        experiment_name=experiment_name, run_name=first_run, config=run_config
+    )
+    # get the first run
+    response = client.get_run(
+        experiment_name=experiment_name, run_name=first_run
+    ).json()
+    assert response['name'] == first_run
+    assert json.loads(response['config']) == run_config
+    # create another run
+    client.create_run(
+        experiment_name=experiment_name, run_name=second_run, config=run_config
+    )
+    # list all runs
+    response = client.list_runs(experiment_name=experiment_name)
+    assert len(response) == 1
+    exp_runs = response[0].json()
+    assert exp_runs[0]['name'] == first_run and exp_runs[1]['name'] == second_run
+    assert (
+        json.loads(exp_runs[0]['config'])
+        == json.loads(exp_runs[1]['config'])
+        == run_config
+    )
+    # delete the first run
+    client.delete_run(experiment_name=experiment_name, run_name=first_run)
+    response = client.list_runs(experiment_name=experiment_name)
+    exp_runs = response[0].json()
+    assert len(exp_runs) == 1
+    assert exp_runs[0]['name'] == second_run
+    # delete all existing runs
+    client.delete_runs(experiment_name=experiment_name)
```

Comment:
after delete run, can we somehow assert the `STATUS` in AWS batch to be inactive or failed? (make sure it is canceled)